backend/routes/gallery/authentication.js

```diff
@@ -0,0 +1,33 @@
+import jwt from "jsonwebtoken";
+import dotenv from "dotenv";
+dotenv.config({ path: '../../.env' });
```

Comment:
we shouldn't specify a specifc env path like this, it defaults to the running directory which is the desired behaviour